.gitignore

```diff
@@ -1,5 +1,4 @@
 *temp*
-assets/
```

Comment:
So I agree with much of what you said. I also had a phone call with @bigmstone where we discussed some of this.

I opened https://github.com/Mierdin/todd/issues/27 to track this because I agree, being able to write these in Go would be nice, but I still absolutely want to preserve the idea that as long as it's executable and it supports the right input/output, it can be a testlet or collector.

I think a future PR should address https://github.com/Mierdin/todd/issues/27, because the elimination of go-bindata, while a good idea, is a larger undertaking than the bullet list item in this PR. So for the time being here's what I propose for this PR in particular:

- Go ahead and include the assets/ directory, as well as the changes to the makefile that allow me to run ```make update_assets```. Given that we're staying with the go-bindata approach for the time being, this at least makes ToDD ```go get```able, which is a good thing.
- Please also include the part of the code that iterates through the assets package and dynamically retrieves objects stored there instead of the gross monstrosity I had before that statically referred to them. Once more, thanks for this.
- I want to make sure that if someone places a testlet in the directory on the server, that the server/assets.go can pick that up and distribute it automatically. So I love the default of serving from the embedded assets, but if you could also serve anything from the filesystem, that would keep things more user friendly until we satisfy https://github.com/Mierdin/todd/issues/27

Hope that makes sense!